src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
         self.__current_operation = None
 
         # Update patch metadata summary in status for auto patching installation requests, to be reported to healthstore
-        if execution_config.maintenance_run_id is not None and execution_config.operation.lower() == Constants.INSTALLATION.lower():
+        if (execution_config.maintenance_run_id is not None or execution_config.health_store_id is not None) and execution_config.operation.lower() == Constants.INSTALLATION.lower():
```

Comment:
I may have missed this in the review. Where is the healthStoreId passed as a parameter to the healthStore metadata function?

Reply:
Its available inside PatchInstaller to write and gets there inside the executionconfig container class...